pre-load.sh

```diff
@@ -7,30 +7,43 @@ do
 
    fqdn=$(gen_fqdn $node)
 
-   scmd $ssh_user@$fqdn <<-\SSH
```

Comment:
We need to retain this as the script is designed to work on a number of remote hosts